site/docs/components/icons.mdx

```diff
@@ -5,13 +5,15 @@ route: /components/icons
 ---
 
 import { Playground } from "docz";
-import { IconFaceSmile } from "hds-react";
+import { IconFaceSmile, StatusLabel } from "hds-react";
```

Comment:
Ei kuulu varsinaisesti tähän pullariin, mutta kokeilin vaihtaa playgroundilla ikonia toiseksi, mutta se ei onnistu, koska on importattu vain yksi ikoni koko ikonikirjaston sijasta.

Reply:
Tätä ei valitettavasti pysty korjaamaan, vaikka importtaisi kaikki HDS:n ikonit kerralla. Playground ei silti dynaamisesti tunnista niitä komponenteiksi. Johtunee siitä, että Gatsby varmaan generoi nuo sivut siinä buildissa.

Reply:
Ok. Jätetään tämä siis näin.